src/main/java/org/geomerty/oop/shapes/Circle.java

```diff
@@ -0,0 +1,17 @@
+package org.geomerty.oop.shapes;
+
+import static java.lang.Math.PI;
+
+public class Circle extends Point {
+    private final double radius;
+
+    public Circle(double x, double y, double radius) {
+        super(x, y);
+        this.x = x;
```

Comment:
@Gortell `this.x = x` is redundant. You've already initialise x with the line above.